Add NVIDIA AI Workbench documentation (#497)
rapidsai/deployment: source/platforms/nvidia-ai-workbench.md

# NVIDIA AI Workbench

> [NVIDIA AI Workbench](https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/workbench/) is a developer toolkit for data science, machine learning, and AI project development that lets you develop on your laptop/workstation and then move workloads to scalable GPU resources in a data center or the cloud. AI Workbench is free, you can install it in minutes on your local or remote computers, and it provides both a desktop application and a command-line interface (CLI).

Reviewer: "AI project development that lets you develop".  This reads a bit redundant, and the first sentence is a bit long, how about

```suggestion
[NVIDIA AI Workbench](https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/workbench/) is a developer toolkit for data science, machine learning, and AI projects. It lets you develop on your laptop/workstation and then easily transition workloads to scalable GPU resources in a data center or the cloud. AI Workbench is free, you can install it in minutes on both local or remote computers, and offers a desktop application  as well as a command-line interface (CLI).
```

## Installation

You can install AI Workbench locally, or on a remote computer that you have SSH access to.

Follow the [AI Workbench installation](https://docs.nvidia.com/ai-workbench/user-guide/latest/installation/overview.html) documentation for instructions on installing on different operating systems.

> ```{admonition} Thick client

Reviewer: The thick/thin client is a concept I wasn't aware of, I had to read both to sort of get it. I wonder if this reduce to GPU on local / GPU on remote. Maybe making this more clear in the titles?

Author: Thanks for calling this out. Reducing jargon is good, I've gone with "Local GPU System" and "Remote GPU System" instead which hopefully adds clarity.

:class: note

> If you are working on a system that has an NVIDIA GPU you can use the AI Workbench installed to set up and configure Docker if you don't have it installed already. Then you can run notebooks in Python environments with access to your GPU.

Reviewer: ```suggestion
If you are working on a system that has an NVIDIA GPU you can use the AI Workbench 
by installing the desktop application and configure Docker if you don't have it installed already. Then you can run notebooks in Python environments with access to your GPU.
```

> To use AI Workbench in this way you only need to install the desktop application.

Reviewer: If we mention to install the desktop app above as I suggest, we don't need this. Feel free to ignore this change though.
```suggestion
```

```

```{admonition} Thin client
:class: note

> If you don't have an NVIDIA GPU in your system, but have remote SSH access to a system that does then you can use AI Workbench to connect to that system. Your code will be executed on the remote system, but files will be synced between your local and remote environments automatically. This allows you to burst from a thin client like a lightweight laptop to a powerful remote AI system.

Reviewer: I think a comma will help here. 
```suggestion
If you don't have an NVIDIA GPU in your system, but have remote SSH access to a system that does, then you can use AI Workbench to connect to that system. Your code will be executed on the remote system, but files will be synced between your local and remote environments automatically. This allows you to burst from a thin client like a lightweight laptop to a powerful remote AI system.
```

> To use AI Workbench in this way you need to install the desktop application on your thin client and the CLI application on the remote system.

Reviewer: ```suggestion
To use AI Workbench in this way you need to install the desktop application on your thin client (local workstation) and the CLI application on the remote system.
```

Select "Start a new project" and give it a name and description. You can also change the default location to store the project files.

![Screenshot of the "Start a new project" button](../_static/images/platforms/nvidia-ai-workbench/create-project.png)

Then scroll down and select "RAPIDS with CUDA" from the list of templates.

![Screenshot of the template selector with "RAPIDS with CUDA" highlighted](../_static/images/platforms/nvidia-ai-workbench/rapids-with-cuda.png)

The new project will then be created. AI Workbench will automatically build a container for this project, this may take a few minutes.

![Screenshot of the AI workbench UI. In the bottom corner the build status says "Building" and the "Open Jupyterlab" button is greyed out](../_static/images/platforms/nvidia-ai-workbench/project-building.png)

Once the project has built you can select "Open Jupyterlab" to launch Jupyter in your RAPIDS environment.

![Screenshot of the AI workbench UI. In the bottom corner the build status says "Build Ready" and the "Open Jupyterlab" button is highlighted](../_static/images/platforms/nvidia-ai-workbench/open-jupyter.png)

> Then you can start working with th RAPIDS libraries in your notebooks.

Reviewer: ```suggestion
Then you can start working with the RAPIDS libraries in your notebooks.
```
